conf/airframes/KS/ks_bebop2_stereo.xml

```diff
@@ -0,0 +1,233 @@
+<!DOCTYPE airframe SYSTEM "../airframe.dtd">
+
+<airframe name="bebop2_opticflow">
+
+  <firmware name="rotorcraft">
+    <target name="ap" board="bebop2"/>
+
+    <module name="telemetry" type="transparent_udp"/>
+    <module name="radio_control" type="datalink"/>
+    <module name="motor_mixing"/>
+    <module name="actuators" type="bebop"/>
+    <module name="imu" type="bebop"/>
+    <module name="gps" type="datalink"/>
+    <module name="stabilization" type="indi_simple"/>
+    <module name="ahrs" type="int_cmpl_quat">
+      <configure name="USE_MAGNETOMETER" value="FALSE"/>
+      <define name="AHRS_USE_GPS_HEADING" value="TRUE"/>
+      <define name="AHRS_HEADING_UPDATE_GPS_MIN_SPEED" value="0"/>
+    </module>
+    <module name="ins" type="extended"/>
+    <!-- Disable use of GPS: -->
+    <!--    <define name="INS_INT_GPS_ID" value="ABI_DISABLE"/>
+    </module> -->
+    
+  </firmware>
+
+  <modules main_freq="512">
```

Comment:
This will be indeed a big help, although we won't enforce this for v5.12 release but hopefully the next one.